### ISSUE TITLE
pipe unrecognized text files

### ISSUE THREAD

**Issue by Konfekt** [wofr06/lesspipe.sh#25]
Opening a file such as `INSTALL` contained in the root directory of this repository by lesspipe.sh returns an empty output. Would it not be more sensible to display this text file anyway?

**Comment by Konfekt** [comment]
One possibility would be to bail out as soon as a text file type is recognized by file that is,
```sh
if [[ $"(file --mime --brief "$filepath")" != "text/*" ]]; then
  cat "$filepath"
  exit
fi
```

**Comment by Konfekt** [comment]
This has been included in afd8a09501a2ad62aa1e83ea256c4d94476f9d1a
